app/javascript/common/UserMessage.js

```diff
@@ -3,11 +3,14 @@ import React from 'react'
 import { Alert } from '@cwds/components'
 
 const UserMessage = ({ errorMsg, successMsg }) => {
+  const successWithEmailChange = Array.isArray(successMsg) ? successMsg[1] : ''
+  const successAlertMessage = !Array.isArray(successMsg) ? successMsg : successMsg[0]
```

Comment:
```suggestion
  const successAlertMessage = Array.isArray(successMsg) ? successMsg[0] : successMsg
```

I think that's equivalent, and much easier to read

Reply:
Updated just now